### PR TITLE
Added file naming options to Settings

### PR DESCRIPTION
**Pull request by LunusLupus**
- Added the ability to add the profession of the PoV to the file name (default:false)
- Added the ability to add the encounter length to the file name (default:false)

### CODE REVIEW

File: LuckParser/Properties/Settings.Designer.cs [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     Ce code a été généré par un outil.
-//     Version du runtime :4.0.30319.42000
+//     Dieser Code wurde von einem Tool generiert.
```
[the remark this answers is not in the document]
Comment (LunusLupus) [reply LuckParser/Properties/Settings.Designer.cs]: Le code allemand a été généré par un outil aussi 
:yum:

File: LuckParser/MainForm.cs [hunk 2]
```diff
@@ -235,6 +235,8 @@ private void BgWorkerDoWork(object sender, DoWorkEventArgs e)
                         throw new CancellationException(rowData, new Exception("Invalid save directory"));
                     }
                     string result = log.LogData.Success ? "kill" : "fail";
+                    string encounterLengthTerm = Properties.Settings.Default.AddDuration ? "_"+(log.LogData.EncounterLength/1000).ToString()+"s" : "";
```
[the remark this answers is not in the document]
Comment (EliphasNUIT) [reply LuckParser/MainForm.cs]: What you could do is add it to ParsedLog, EncounterDuration => FightData.FightDuration, would be cleaner I think

[the remark this answers is not in the document]
Comment (EliphasNUIT) [reply LuckParser/MainForm.cs]: No leave it like that, I have a better idea, the whole LogData object is useless

**Comment by EliphasNUIT** [comment]
Could you add your 2 settings to sample.conf please? So that console users know it exists

**Pull request merged**